### PR TITLE
Add whitelisting prerequisites

### PR DESCRIPTION
**Pull request by tbruno**

### CODE REVIEW

**Comment by dosten** [comment]
In order to be able to install Service Mesh we also need to whitelist:

- https://storage.googleapis.com (to download kubectl)
- https://get.helm.sh (to download helm)

**Review by dosten — APPROVED** [review]
LGTM

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -64,6 +64,27 @@ Before you can use Anypoint Service Mesh, you must have the following roles and
...
+|===
+|Plane |Host |Port
+|*US*|*anypoint.mulesoft.com* | 443
+|*US*|*analytics-ingest.anypoint.mulesoft.com* |  443
+|*US*|**.dkr.ecr.us-east-1.amazonaws.com* |  443
+|*US*|*exchange-files.anypoint.mulesoft.com* |  443
+|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443
+|*EU*|*eu1.anypoint.mulesoft.com* | 443
+|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443
+|*EU*|**.dkr.ecr.eu-central-1.amazonaws.com* |  443
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc:81]: I'm not sure what the double asterisk here and on line 76 mean, but I assume they're intentional.

**Review by wwarner-salesforce — APPROVED** [review]

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (dosten) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc:81]: It's fine, we need to whitelist any subdomain under `dkr.ecr.eu-central-1.amazonaws.com` so `*.dkr.ecr.eu-central-1.amazonaws.com` it's ok IMO.

**Pull request merged**